prow/crier/reporters/gerrit/reporter.go

```diff
@@ -309,19 +316,27 @@ func ParseReport(message string) *JobReport {
 	if !isReport {
 		return nil
 	}
-	report := &JobReport{}
-	report.header = contents[start]
-	for i := start; i < len(contents); i++ {
-		j := deserializeJob(contents[i])
-		if j != nil {
-			report.Jobs = append(report.Jobs, j)
+	var report JobReport
+	report.Header = contents[start] + "\n"
+	for i := start + 1; i < len(contents); i++ {
+		if contents[i] == "" {
+			continue
 		}
+		var j Job
+		if err := deserialize(contents[i], &j); err != nil {
+			logrus.Warnf("Could not deserialize %s to a job: %v", contents[i], err)
```

Comment:
add a `continue` ?
i.e. if deserialize failed we shouldn't add it to the total, success, jobs list.